octobot_trading/personal_data/orders/order_state.py

```diff
@@ -77,6 +83,12 @@ async def should_be_updated(self) -> bool:
         """
         return self.order.is_self_managed()
 
+    async def change_order(self, new_order):
```

Comment:
I don't really like the word `change` here, I think `replace` would be better. What do you think?

Reply:
right